engine-comparison/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM gcr.io/fuzzer-test-suite/clang-deps-image
+
+RUN apt-get update
+
+# Create directory, and set it as the working directory "."
+WORKDIR /~/work
```

Comment:
Sure, but why the leading `/`?
At least on my machine, `ls /~` yields `ls: cannot access /~: No such file or directory`.